Terasology/WorkstationCrafting
# Migration of wood and stone-based processes and items to new modules

After some extensive discussions with @Cervator and a few other members on Slack, we've decided to reorganize part of the WorkstationCrafting (WC)/Neo Throughout the Ages (NeoTTA) family of modules. 

### Contains

The first step of this was to divide WorkstationCrafting into three modules: one that was a systems-focused module that extended the base Workstation processes and systems (the new WC), one that was a content-module which focused on Woodcrafting, and one that was a content module which focused on Stonecrafting. This was to reduce the number of dependencies present in WC, and make it easier to extend. Thus, the crafting on hand and wood-related items/processes were migrated over to [Woodcrafting](https://github.com/Terasology/Woodcrafting), and the milling and stone-related processes were migrated over to [Stonecrafting](https://github.com/Terasology/Stonecrafting).

Each module further in terms of technological development would depend on the previous one. So, Stonecrafting depends on Woodcrafting, which itself depends on WorkstationCrafting. In the future, CopperAndBronze will be updated to depend on Stonecrafting.

### How to test

* See if all the item prefabs can be spawned using the `give` command.
* Download the [Woodcrafting](https://github.com/Terasology/Woodcrafting) and [Stonecrafting](https://github.com/Terasology/Stonecrafting) modules, and test to ensure all the items and processes in those modules can be successfully created.

### Issues

* The special recipe prefabs defined in RegisterWoodcraftingRecipes and RegisterStonecraftingRecipes can't be made due to an NPE.

### Credits

@Cervator, @skaldarnar, @skysom, and two other members for discussing how to rework the NeoTTA family of modules.
